tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2696,6 +2696,26 @@ def assignDerivedParameters(state):
     if state["LdsBlockSizePerPadMetadata"] == -1:
       state["LdsBlockSizePerPadMetadata"] = state["LdsBlockSizePerPadA"]
 
+    if state["LdsBlockSizePerPadA"] != 0 :
```

Comment:
If the value of ldsBlockSizePerPadA is set by user, we can reject it.
- Reject section can be here.
- Setting as 0 section should be in the handling of LdsBlockSizePerPad = -1.

Reply:
yes, if the user sets LdsBlockSizePerPadA = -1 will set LdsBlockSizePerPadA to 0, otherwise will be rejected.

          if auto_LdsBlockSizePerPadA_for_mix:   # LdsBlockSizePerPadA=-1
            print("Paded addres is inconisstent, set LdsBlockSizePerPadA=0.")
            state["LdsBlockSizePerPadA"] = 0
            break
          else:
            reject(state, "A's paded addres is inconisstent")